products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,179 @@
+
+
+### Data Model
+
+* Each Football transfer has a team which makes transfere of player and 
+team to which player is transfered to, so we identify three types of nodes &mdash;
+`Team`, `Transfer` and `Player`  . 
+* We add an edge of type `:TransFor` pointing from `Team` player is being
+transfered from to a `Transfer`. Also, we add an edge of type `:TransTo` pointing
+from the `Transfer` to the `Team` player is being transfered to.
+From node `Transfer` we add edge of type `:OfPlayer` pointing to `Player` 
+* Each team has a name so we can add property `name` to `Team` node.
+* We'll add properties `name` and `position` to node `Player`. 
+* And to node `Transfer` we wil add properties `age` and `fee` with
+property `age` representing age at which player was transfered to
+and fee for which player was transfered to. 
+* Each transfer is made in specific year, so we can
+create node `Year` with property `year` and relationship `:InYear` between
+transfer and year.
+
+
+![](../data/FootballTransfers_metagraph.png)
+
+### Importing the Snapshot
+
+TO DO...
+
+### Example Queries using OpenCypher
+
+In the queries below, we are using [OpenCypher](https://www.opencypher.org) 
+to query Memgraph via the console.
+
+
+1) Let's say you want to find 20 most expensive transfers
+
+```opencypher
+MATCH (t:Transfer)-[:ofPlayer]->(p:Player)
+WHERE t.fee is NOT NULL
+RETURN  DISTINCT t.fee ,p.name 
+ORDER BY t.fee DESC LIMIT 20
+```
+
+2) Find all clubs for which player has played
+
+```opencypher
+MATCH (p:Player {name:"Neymar"})<-[:ofPlayer]-(t:Transfer)<-[:makesTransfer]-(m:Team) 
+RETURN DISTINCT m.name as name
+UNION 
+MATCH (p:Player {name:"Neymar"})<-[:ofPlayer]-(t:Transfer)-[:toClub]->(n:Team) 
+RETURN DISTINCT n.name as name
+```
+
+3)Find players that played for certain club and count them by position.
+
+```opencypher
+MATCH (m:Team {name: "FC Barcelona"})
+      <-[:toClub]-(t:Transfer)-[:ofPlayer]->(p:Player)
+WITH * ORDER BY p.name
+RETURN p.position, COUNT(t) AS number
+```
+
+4) Transfers of players directly from on club to another
+
+```opencypher
+MATCH (m:Team {name:"FC Barcelona"})-[r1:makesTransfer]->(t:Transfer)
+MATCH (t)-[r3:ofPlayer]->(p:Player)
+MATCH (t)-[r4:inYear]->(y:Year)
+MATCH (t)-[r2:toClub]->(n:Team {name:"Real Madrid"})
+RETURN m,r1,t,r2,n,p,r3,r4,y
+```
+
+5) Get all clubs involved and transfers of player in graph style
+
+```opencypher
+MATCH p= (a:Team )
+          -[:makesTransfer ]->(t:Transfer)-[:ofPlayer ]->
+          (:Player {name: "Zlatan Ibrahimovic"})
+MATCH (t)-[r1:toClub]->(b:Team) 
+return p,b,r1
+```
+
+6) Find teams that  most players went to from let's say "FC Barcelona"
+
+```opencypher
+MATCH (m:Team {name: "FC Barcelona"})-[:makesTransfer]->(t:Transfer)-[:ofPlayer]->(p:Player)
+MATCH (t)-[:toClub]->(n:Team)
+RETURN n.name, collect(p.name), COUNT(p) AS numberOfPlayers
+ORDER BY numberOfPlayers DESC
+```
+
+
+7) Find most popular clubs in certain year. Results may suprise you.
+
+```opencypher 
```

Comment:
I won't be adding the same comments from this point on, but in general for every query:
- Do a bit of cleanup (consistency, variable naming)
- Add cleaned edge types, UPPER_SNAKE_CASE
- Get rid of unnecessary commands
- If the image makes sense, add an image from Lab with link to the lab
- If the query is numbered with number 7 or higher, add a bit of description (check other tutorials for reference)